thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/dto/EventDTO.java

```diff
@@ -2,6 +2,8 @@
 
 import com.linkedin.thirdeye.datalayer.pojo.EventBean;
 
+
```

Comment:
you could revert these whitespace changes to make the PR smaller

Reply:
Sure